recipes/pixman/all/conanfile.py

```diff
@@ -92,22 +106,20 @@ def build(self):
             autotools.make(target="pixman")
```

Comment:
by the way, this target is useless because install target is called later, building everything (tests etc) during package() step.

Reply:
Good point :smile: